MovieQuiz/Presentation/MovieQuizViewController.swift

```diff
@@ -2,11 +2,197 @@ import UIKit
 
 final class MovieQuizViewController: UIViewController {
     // MARK: - Lifecycle
+    
+    @IBOutlet weak private var counterLabel: UILabel!
+    @IBOutlet weak private var imageView: UIImageView!
+    @IBOutlet weak private var textLabel: UILabel!
+    
+
+
+    private var currentQuestionIndex: Int = 0
```

Comment:
Принято оставлять максимум одну пустую строку между логическими блоками
```suggestion

    private var currentQuestionIndex: Int = 0
```